pulp_ansible/tests/unit/test_search.py

```diff
@@ -0,0 +1,157 @@
+import hashlib
+import itertools
+import os
+import random
+import string
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.db import connection
+from django.test import TestCase
+
+from pulp_ansible.app.models import (
+    AnsibleDistribution,
+    AnsibleRepository,
+    Collection,
+    CollectionVersion,
+    Tag,
+)
+from pulp_ansible.tests.unit.helpers import make_cv_tarball
+from pulpcore.plugin.models import Artifact, ContentArtifact
+
+
+class TestSearchUtil(TestCase):
+
+    repo = None
+    collections = None
+
+    def setUp(self):
+
+        # make a couple repositories to distrbute the collections around
+        #   this is also to prepare for future cross-repo searching
+        self.repos = {}
```

Comment:
After reading through the test, I don't see any use of these repos or their distributions. I think these would better suited for use in a functional test as this unit test is checking that the database field is correct, so I suggest these should be removed.

Reply:
I copied the bulk of this test from one that I'm working on for the multi-repo search feature, so yeah there's a lot of cruft I can remove for this specific test. I'm still not 100% on what model/objects need to get created for these workflows, so I was hoping for reviews like this to point out what I didn't need to add.